src/Http/Request/Body/JsonSchemaValidatingRequestBodyParser.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Request\Body;
+
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\ApiProblem\SchemaError;
+use Opis\JsonSchema\Errors\ErrorFormatter;
+use Opis\JsonSchema\Validator;
+use Psr\Http\Message\ServerRequestInterface;
+
+final class JsonSchemaValidatingRequestBodyParser implements RequestBodyParser
+{
+    private const SCHEMA_LOCATIONS = __DIR__ . '/../../../../vendor/publiq/stoplight-docs-uitdatabank/models/';
+    private const MAX_ERRORS = 100;
```

Comment:
I opted for private constants here for now because there's no real use case to make them configurable per instance now. Only downside is that this couples this code a bit to how the `publiq/stoplight-docs-uitdatabank` package is structured but for loading the schemas from the filesystem I think that is not a really big problem and it makes it a lot easier to instantiate this class this way.

Reply:
I made a remark about the tight coupling here https://github.com/cultuurnet/udb3-silex/pull/703#discussion_r690993746